code/modules/chemistry/Chemistry-Recipes.dm

```diff
@@ -4013,4 +4013,14 @@ datum
 			required_reagents = list("poor_cement" = 1, "silicon_dioxide" = 5, "water" = 1)
 			result_amount = 7
 
+		hot_toddy
+			name = "Hot Toddy"
+			id = "hottoddy"
+			result = "hottoddy"
+			required_reagents = list("bourbon" = 1, "tea" = 1, "juice_lemon" = 1)
```

Comment:
Consider adding `sugar` to this (or `honey`, but I don't think we have that in reagent form in any easily obtainable way), to be a bit truer to the actual recipe?

Reply:
Honey is obtainable from bees. Botany will usually have a surplus